compose/project.py

```diff
@@ -649,7 +649,7 @@ def up(self,
            ):
 
         if cli:
-            log.warning("Native build is an experimental feature and could change at any time")
+            log.info("Building with native build; set COMPOSE_DOCKER_CLI_BUILD=0 to disable")
```

Comment:
Reading this quickly and it seems as if compose is telling me I **should** disable native build. Could we turn it around a little bit and make it more explicit that they don't have to if they don't want?

Reply:
Good one. My intent was to make it easy to discover how to switch, but (I guess) env-vars being UPPERCASE makes it less subtle and feel more like "you should do this".

Should we include a URL with details on how to switch?

```
Building with native build. Learn about native build in compose here: https://docs.docker.com/go/compose-native-build
```

Reply:
+1 to adding a link to the docs!

One nit: `Building with native build. Learn about native build in Compose here`

Reply:
I agree! Let's make sure we use re-directs again or something here so we can track the number of people this is impacting :)/who are using the link from the CLI

Reply:
I updated the message; https://github.com/docker/compose/pull/7978/files and a PR for the documentation is opened in https://github.com/docker/docker.github.io/pull/11858, but likely would have to wait being merged until / just before this patch is released and included in Docker Desktop (as it mentions the new default)